app/views/state_file/questions/nj_disabled_exemption/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+    title = t(".title", filing_year: @filing_year)
```

Comment:
[dust] Are we using filing year in this page title? If not it could be removed here

Reply:
No! Thanks for catching that. Fixed.